chips/virtio/Cargo.toml

```diff
@@ -0,0 +1,8 @@
+[package]
+name = "virtio"
+version.workspace = true
+authors.workspace = true
+edition = "2018"
```

Comment:
we have been starting new crates at edition 2021 for awhile now

Reply:
Great catch, thanks!